### PR TITLE
UI update/update delegation

### PR DESCRIPTION
**Pull request by soerenbf**
## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: packages/browser-wallet/src/popup/popupX/pages/EarningRewards/Delegator/TransactionFlow.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                return;
+            }
+
+            nav(pathname, { replace: true, state: form }); // Override current router entry with stateful version
+
+            const submitDelegatorState: DelegationResultLocationState = {
+                payload,
+                type: existingValues !== undefined ? 'change' : 'register',
+            };
+            nav(absoluteRoutes.settings.earn.delegator.submit.path, { state: submitDelegatorState });
```
Comment (limemloh) [inline packages/browser-wallet/src/popup/popupX/pages/EarningRewards/Delegator/TransactionFlow.tsx:62]: **Question:** I get an error when submitting is this expected?

The error is related to signing it seems `Uint8Array expected`

File: packages/browser-wallet/src/popup/popupX/pages/EarningRewards/Delegator/TransactionFlow.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+import { configureDelegatorPayloadFromForm, type DelegatorForm } from './util';
+import { DelegationResultLocationState } from './Result/DelegationResult';
+
+function NoChangesNotice(props: FullscreenNoticeProps) {
+    const { t } = useTranslation('x', { keyPrefix: 'earn.delegator.update.noChangesNotice' });
+    return (
+        <FullscreenNotice {...props}>
+            <Page>
+                <Page.Top heading={t('title')} />
+                {t('description')}
```
Comment (limemloh) [inline packages/browser-wallet/src/popup/popupX/pages/EarningRewards/Delegator/TransactionFlow.tsx:25]: **Suggestion:** Wrap in some text component, the font seems to be different at least.

File: packages/browser-wallet/src/popup/popupX/pages/EarningRewards/Delegator/TransactionFlow.tsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+import Button from '@popup/popupX/shared/Button';
+import DelegatorStake from './Stake';
+import DelegatorType from './Type';
+import { configureDelegatorPayloadFromForm, type DelegatorForm } from './util';
+import { DelegationResultLocationState } from './Result/DelegationResult';
+
+function NoChangesNotice(props: FullscreenNoticeProps) {
+    const { t } = useTranslation('x', { keyPrefix: 'earn.delegator.update.noChangesNotice' });
+    return (
+        <FullscreenNotice {...props}>
```
Comment (limemloh) [inline packages/browser-wallet/src/popup/popupX/pages/EarningRewards/Delegator/TransactionFlow.tsx:22]: **Question (non-blocking):** I find the flow hard to follow since some of it is tracked in the path and some in modals, why is this even a modal and not just another page part of the routing? :smile:

**Review by limemloh — APPROVED** [review]

File: packages/browser-wallet/src/popup/popupX/pages/EarningRewards/Delegator/TransactionFlow.tsx (hunk 3 above)
Comment (soerenbf) [reply packages/browser-wallet/src/popup/popupX/pages/EarningRewards/Delegator/TransactionFlow.tsx:22]: The point of these "notices" is to have the user confirm some action or simply show some information to the user. Routing to another page tears down the state built for that route/page, which would require you to save the state somehow to allow the user to return to the previous page and rebuild the page from the saved state.

We can completely avoid this by treating this as what it actually is (a prompt for the user) instead of what it looks like (a subsequential page in the flow) 😄

File: packages/browser-wallet/src/popup/popupX/pages/EarningRewards/Delegator/TransactionFlow.tsx (hunk 2 above)
Comment (soerenbf) [reply packages/browser-wallet/src/popup/popupX/pages/EarningRewards/Delegator/TransactionFlow.tsx:25]: You're right, good catch 👍
